tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -27,20 +38,65 @@ public void run() {
     public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
              final var outputStream = connection.getOutputStream()) {
+            HttpRequest httpRequest = new HttpRequest(inputStream);
 
-            final var responseBody = "Hello world!";
-
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
+            log.info("[ REQUEST START LINE : {} ]", httpRequest.getStartLine());
 
-            outputStream.write(response.getBytes());
-            outputStream.flush();
+            if (httpRequest.getUri().equals("/")) {
+                returnIndexPage(outputStream);
+                return;
+            }
+            if (httpRequest.isStaticResourceRequest()) {
+                returnStaticResource(outputStream, httpRequest);
+                return;
+            }
+            if (httpRequest.isQueryString()) {
+                Map<String, String> queryString = httpRequest.parseQueryString();
+                Optional<User> user = InMemoryUserRepository.findByAccount(queryString.get("account"));
+                if (!user.isEmpty() && user.get().checkPassword(queryString.get("password"))) {
```

Comment:
```suggestion
                if (user.isPresent() && user.get().checkPassword(queryString.get("password"))) {
```
를 쓰거나 Optional에서 `.orElseThrow()`를 사용해보는 것도 좋겠네요!